.github/container/install-pax.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+## Parse command-line arguments
+
+usage() {
+    echo "Usage: $0 [OPTION]..."
+    echo "  -d, --dir=PATH         Path to store Pax source. Defaults to /opt"
+    echo "  -f, --from_paxml=URL   URL of the Paxml repo. Defaults to https://github.com/google-research/paxml.git"
+    echo "  -g, --from_praxis=URL  URL of the Praxis repo. Defaults to https://github.com/google-research/praxis.git"
```

Comment:
It's better if we use short option letters that are related to the long options. For example, `-h` for `--help`. If there is no good candidate for a short option, it is OK to only define the long option.